sklearn/mixture/gmm.py

```diff
@@ -64,7 +64,8 @@ def log_multivariate_normal_density(X, means, covars, covariance_type='diag'):
     return log_multivariate_normal_density_dict[covariance_type](
         X, means, covars)
 
-
+@deprecated("The class function sample_gaussian is deprecated and "
+            "will be removed in 0.20. Use numpy.multivariate_normal instead.")
```

Comment:
If you do git grep sample_gaussian, you'll see that the class _GMM uses it.
So we need to create a _sample_gaussian function.